man/os-release.xml

```diff
@@ -79,6 +81,19 @@
     <para>For a longer rationale for <filename>os-release</filename>
     please refer to the <ulink
     url="http://0pointer.de/blog/projects/os-release">Announcement of <filename>/etc/os-release</filename></ulink>.</para>
+
+    <refsect2>
+      <title><filename>/etc/initrd-release</filename></title>
+
+      <para>In the <ulink
+      url="https://www.kernel.org/doc/html/latest/admin-guide/initrd.html">initramfs</ulink>,
```

Comment:
we never call this "initramfs" actually, always "initrd"

(both are technically wrong, since today it's actually backed by tmpfs iirc, but the name "initrd" simply stuck the best)

Reply:
Updated.